tests/nvme_test.py

```diff
@@ -228,6 +231,29 @@ def get_ncap(self):
         print(ncap)
         return int(ncap)
 
+    def get_ocfs(self):
+        """ Wrapper for extracting optional copy formats supported
+            - Args:
+                - None
+            - Returns:
+                - Optional Copy Formats Supported
+        """
+        pattern = re.compile("^ocfs[ ]+: 0x[0-9a-fA-F]+", re.IGNORECASE)
+        ocfs = 0
+        max_ns_cmd = "nvme id-ctrl " + self.ctrl
```

Comment:
Name doesn't really make sense

Reply:
I think this is based off of another similar function.

Reply:
Oh, I see. Yes, it shouldn't be `max_ns_cmd`.